web/models.py

```
@@ -2581,6 +2581,98 @@
+
+class PDFSubmission(models.Model):
+    """Model for storing PDF documents submitted by students for review."""
+
+    STATUS_CHOICES = [
+        ("submitted", "Submitted"),
+        ("under_review", "Under Review"),
+        ("reviewed", "Reviewed"),
+        ("needs_revision", "Needs Revision"),
+        ("approved", "Approved"),
+        ("rejected", "Rejected"),
+    ]
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)